### ISSUE TITLE
Video stops playing after a while, with no console error

### ISSUE THREAD

**Issue by nathaliesicard** [ronik-design/react-drive-in#31]
Video doesn't loop.

**Comment by nathaliesicard** [comment]
var React = require('react');
const ReactDriveIn = require("react-drive-in");

var bgImgSrc = require('../assets/images/bg.jpg');

var bgVideoSrc = require('../assets/videos/video-5.mp4');

var Main = React.createClass({

  render: function() {
    return (
      <div>
        <ReactDriveIn
          show={bgVideoSrc}
          poster={bgImgSrc}
          loop
        />
        {this.props.children}
      </div>
    );
  }
});

module.exports = Main;

**Comment by mshick** [comment]
I don't think you would require the assets like that. Just provide the public paths to them as strings, e.g. `/assets/videos/video-5.mp4`. 

To be honest, I'm not sure what requiring files like that would do... I guess it would depend on your build system? Maybe you're doing something with webpack loaders here? Perhaps you can explain more, or, provide the generated snippet from your build system. Or console errors would be helpful.

**Comment by mshick** [comment]
Oh, just re-read the title -- you said, no console error! Definitely curious. Any extra info you can provide would be super helpful.

**Comment by nathaliesicard** [comment]
I don't know what's going on. I thought it was something wrong with this plugin. And then I rewrote it in my own way (notice I am a totally beginner so my code might not be great). And it happens exactly the same. I think it might be something to do with Chrome. Haven't been able to solve it.
You can check out my repository https://github.com/nathaliesicard/yoguin
